internal/support/cronscheduler/infrastructure/interfaces/db.go

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package interfaces
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v3/errors"
+	model "github.com/edgexfoundry/go-mod-core-contracts/v3/models"
+)
+
+type DBClient interface {
+	CloseSession()
+
+	AddScheduleJob(scheduleJob model.ScheduleJob) (model.ScheduleJob, errors.EdgeX)
+	AllScheduleJobs(start, end, offset, limit int) ([]model.ScheduleJob, errors.EdgeX)
```

Comment:
always use int64 for timestamp, such as start and end, to prevent issues from arm32 system